docs/annotation.md

```diff
@@ -125,27 +125,81 @@ Many remote sensing assets are stored as an ImageServer within ArcGIS REST proto
 
 More work is needed to encompass the *many* different param settings and specifications. We welcome pull requests from those with experience with  [WebMapTileServices](https://enterprise.arcgis.com/en/server/latest/publish-services/windows/wmts-services.htm).
 
-## Example: Specify a lat-long box and crop an ImageServer asset 
+## Example 1: Specify a lat-long box and crop an ImageServer asset 
```

Comment:
Can you delete 'Example 1' and 'Example 2' and just use the text heading "Specifiy a lat-long box..." and "Downloading a batch of images"